examples/sites/demos/pc/menus.js

```diff
@@ -175,6 +175,7 @@ export const cmpMenus = [
       { 'nameCn': '输入框', 'name': 'Input', 'key': 'input' },
       { 'nameCn': ' IP地址输入框', 'name': 'IpAddress', 'key': 'ip-address' },
       { 'nameCn': '数字输入框', 'name': 'Numeric', 'key': 'numeric' },
+      { 'nameCn': '标签输入框', 'name': 'TagInput', 'key': 'tag-input' },
```

Comment:
参考文档：[官网组件以及API版本状态维护信息管理方案](https://github.com/opentiny/tiny-vue/wiki/%E5%AE%98%E7%BD%91%E7%BB%84%E4%BB%B6%E4%BB%A5%E5%8F%8AAPI%E7%89%88%E6%9C%AC%E7%8A%B6%E6%80%81%E7%BB%B4%E6%8A%A4%E4%BF%A1%E6%81%AF%E7%AE%A1%E7%90%86%E6%96%B9%E6%A1%88)

添加组件 meta 信息。

```suggestion
      {
        'nameCn': '标签输入框', 'name': 'TagInput', 'key': 'tag-input',
        'meta': {
            'experimental': '3.29.0'
          }
      },
```